### DISCUSSION TITLE
Excel formulas are not calculated (Laravel 10.22)

### DISCUSSION THREAD

**Discussion by xGrz** [spatie/simple-excel#151] [Q&A]
When importing an Excel file, formulas (addition, multiplication) are not calculated. After import, they look exactly as they are entered in Excel (e.g. =D2*1.23). I did some investigation and it looks like it might be a Laravel dependency issue. Could someone check this for themselves and confirm that simple-excel does not calculate values on import for the latest version of Laravel (10.22)?

**Comment by xGrz** [comment]
After investigation, update the _openspout/openspout_ dependency from 4.16.0 to 4.17.1. prevents calculation of Excel cell values. Can I count on a patch?

**Reply by bobmulder** [reply]
I think making a PR would help the guys at spatie and you to get this fixed ;)

**Reply by freekmurze** [reply]
Feel free to send a PR!
